tests/acceptance/test_internationalization.py

```diff
@@ -69,3 +70,63 @@ def test___supported_encoding___logging_file_path___returns_assigned_value(
     ai_task.in_stream.logging_file_path = file_path  # type: ignore[assignment] # https://github.com/ni/nidaqmx-python/issues/613
 
     assert ai_task.in_stream.logging_file_path == pathlib.Path(file_path)
+
+
+@pytest.mark.grpc_xfail(
+    reason="AB#2393811: DAQmxGetLoggingFilePath returns kErrorNULLPtr (-200604) when called from grpc-device.",
```

Comment:
FYI: This was fixed in March. Separate from this PR, we should update the version of DAQmx installed on nidaqmxbot and remove xfails referencing this bug.

Reply:
Internal Work Item #2805031